indico/modules/events/sessions/operations.py

```diff
@@ -0,0 +1,63 @@
+# This file is part of Indico.
+# Copyright (C) 2002 - 2015 European Organization for Nuclear Research (CERN).
+#
+# Indico is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 3 of the
+# License, or (at your option) any later version.
+#
+# Indico is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Indico; if not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import unicode_literals
+
+from flask import session
+
+from indico.core.db import db
+from indico.modules.events.sessions import logger
+from indico.modules.events.sessions.models.sessions import Session
+from indico.modules.events.logs.models.entries import EventLogRealm, EventLogKind
+
+
+def create_session(event, data=None):
```

Comment:
I still think the `=None` here is useless (same for the `or {}` in the function)